### PR TITLE
Feature/player controller

### PR DESCRIPTION
**Pull request by ikeinyyo**

### CODE REVIEW

**Comment by danimart1991** [comment]
I can't approve a two commit pull request.

File: src/Assets/Scripts/ShotController.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
+﻿using UnityEngine;
+
+public class ShotController : MonoBehaviour
+{
+    public Material Material;
+    public float MaxDistance = 10;
+
+    private LineRenderer line;
```
Comment (danimart1991) [inline src/Assets/Scripts/ShotController.cs]: private fields must be over properties.

File: src/Assets/Scripts/ShotController.cs.meta [hunk 2]
```diff
@@ -0,0 +1,12 @@
+fileFormatVersion: 2
```
Comment (danimart1991) [inline src/Assets/Scripts/ShotController.cs.meta]: Must meta files be uploaded?

File: src/Library/AssetImportState [hunk 3]
```diff
@@ -1 +0,0 @@
-5;0;6;0;0
```
Comment (danimart1991) [inline src/Library/AssetImportState]: WTF?

**Review by CarlosKlsOne — APPROVED** [review]

**Pull request merged**